### PR TITLE
So Many Things

### PR DESCRIPTION
**Pull request by connorwinston**

### CODE REVIEW

File: app/components/content/ProsePre.vue [hunk 1]
```diff
@@ -54,12 +50,22 @@ withDefaults(
...
 }
 pre {
-	width: 100%;
 	font-weight: normal;
 	padding: 0.7rem 1rem;
+
+	word-wrap: normal;
+	white-space: pre;
+	position: relative;
+	overflow: scroll;
```
Comment (hanneskuettner) [inline app/components/content/ProsePre.vue]: Should be 

```suggestion
	overflow: auto;
```

File: app/components/content/ProsePre.vue [hunk 2]
```diff
@@ -54,12 +50,22 @@ withDefaults(
...
+}
+
+.prose-pre {
 	border-radius: var(--border-radius);
 	background: var(--background-subdued);
 	border: 1px solid var(--border-subdued);
+	position: relative;
+	overflow: scroll;
```
Comment (hanneskuettner) [inline app/components/content/ProsePre.vue]: Should be 

```suggestion
	overflow: auto;
```

File: app/components/content/ProsePre.vue (hunk 1 above)
Comment (br41nslug) [reply app/components/content/ProsePre.vue]: This change will prevent all the scrollbars from showing up unneccesarily
image

File: .github/workflows/lint-app.yaml [hunk 3]
```diff
@@ -14,6 +14,7 @@ concurrency:
 
 env:
   NODE_OPTIONS: --max_old_space_size=6144
+  NUXT_PUBLIC_PRODUCT_DIRECTUS_URL: https://product-team.directus.app
```
Comment (br41nslug) [inline .github/workflows/lint-app.yaml:17]: `NUXT_PUBLIC_PRODUCT_DIRECTUS_URL` will also need to be renamed from `NUXT_PRODUCT_DIRECTUS_URL` in the `example.env` file

**Review by br41nslug — COMMENTED** [review]
Couple of notes i couldnt directly find a good place for in the code changes:

1. overlapping icons and labels on the api reference sidebar (in Chrome/light mode)
image

2. This "collection names are immutable" block seems to have light styling in dark mode
image

3. The top and side navigations are not sticky which is what i would expect but that may be intentional

4. The top most banner and header navigation are not within a max-width container like the rest of the site (I would expect all 3 to be either full width or containered having 2 out of 3 looks a bit misaligned on larger screens imo)
image

5. Last but not least there are a ton of accessibility improvements to be gained in keyboard tab order and contrast in both light and dark mode. For example blue text on blue backgrounds, grey text on grey, white text on light purple, white text on light pink and so on (not sure if WCAG has been discussed for the docs)

**Comment by hanneskuettner** [comment]
Is there any chance we can make the sidenav on the left and the TOC on the right sticky, similar to what supabase does in their docs (https://supabase.com/docs/guides/database/overview) and only switch out the actual page content through a `<router-view/>`/ a common nuxt layout?

Right now navigating between pages feels super unergonomic and is not a nice experience.

**Pull request merged**